AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -23,6 +23,7 @@
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
 chiebukuro.yahoo.co.jp###app_bnr
+||point.rakuten.co.jp/img/crossuse/footer-banner/
```

Comment:
I see it only on mobile, please move the rule to the directive.

Reply:
Not a big problem. Anyway.